packages/apps/human-app/server/src/modules/jobs-discovery/jobs-discovery.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Headers, Query } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+import { JobsDiscoveryService } from './jobs-discovery.service';
+import {
+  JobsDiscoveryParamsCommand,
+  JobsDiscoveryParamsDto,
+  JobsDiscoveryResponse,
+} from './interfaces/jobs-discovery.interface';
+
+@Controller()
+export class JobsDiscoveryController {
+  constructor(
+    private readonly jobsDiscoveryService: JobsDiscoveryService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  @ApiTags('Jobs-Discovery')
+  @Get('/jobs')
+  @ApiOperation({
+    summary:
+      'Retrieve a list of filtered available jobs for passed Exchange Oracle url',
+  })
+  public async getJobs(
+    @Query() jobsDiscoveryParamsDto: JobsDiscoveryParamsDto,
+    @Headers('authorization') token: string,
+  ): Promise<JobsDiscoveryResponse> {
```

Comment:
It should not request for Ex Oracle url, only its address. The url should be obtained from KVStore with the key "url"